mithril-common/src/entities/signer.rs

```diff
@@ -32,6 +34,26 @@ impl Signer {
     }
 }
 
+impl PartialOrd for Signer {
```

Comment:
I do not get the business meaning of implementing a `Ord` on Signer. If it is for display purpose, it would be better the handle this on the display side, what do you think?

Reply:
This ensures that the `Certificate` hash is computed deterministically, which is not the case if the order is not guaranteed